### PR TITLE
[Fix bug 1183220] Fix privacy_level handling on es results.

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

**Comment by johngian** [comment]
@glogiotatidis r?

**Comment by johngian** [comment]
or @akatsoulas r?

File: mozillians/users/tests/test_models.py [hunk 1]
```diff
@@ -187,6 +187,16 @@ def test_extract_document(self):
...
 
+    def test_privacy_aware_iterator(self):
+        UserFactory.create(userprofile={'ircname': 'foo'})
+
+        s = PrivacyAwareS(UserProfileMappingType)
+        s.privacy_level(PUBLIC)
+        q = s.query(ircname='foo')
+
+        eq_(len(q), 1)
+        eq_(q[0]._privacy_level, PUBLIC)
```
Comment (glogiotatidis) [inline mozillians/users/tests/test_models.py]: There is something I don't get about this test: 
 - The created profile is not public
 - You're making a public query and you get the object

Comment (johngian) [inline mozillians/users/tests/test_models.py]: Sounds good! @akatsoulas ?

**Comment by akatsoulas** [comment]
Tested locally and looks good. r+ from me too :beers:

**Pull request merged**